src/app/client/src/app/modules/shared/components/redirect/redirect.component.ts

```diff
@@ -0,0 +1,30 @@
+import { ResourceService } from '../../services';
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-redirect',
+  templateUrl: './redirect.component.html',
+  styleUrls: ['./redirect.component.css']
+})
+/**
```

Comment:
Typos in this comment.

'Redirectcomponent is called ...' change to 'Redirect component is called ...'
'lean/redict' change to 'learn/redirect'

Reply:
done